### DISCUSSION TITLE
Bioinformatics

### DISCUSSION THREAD

**Discussion by Moxiemiya** [community/community#55572] [Programming Help]
How can I use biopython for FASTA analysis?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Moxiemiya ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
